packages/moonstone/VideoPlayer/MediaControls.js

```diff
@@ -405,7 +405,7 @@ const MediaControlsBase = kind({
 
 		return (
 			<OuterContainer {...rest} spotlightId={spotlightId}>
```

Comment:
Could we have assigned `spotlightDisabled` to this container rather than wrapping left/right components with a new container?